DancingGoat/Infrastructure/SelfConfigActionFilter.cs

```diff
@@ -1,25 +1,34 @@
 ﻿using System;
-using System.Web.Mvc;
+using DancingGoat.Areas.Admin;
 using DancingGoat.Areas.Admin.Models;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.Web.CodeGeneration.Contracts.Messaging;
 
 namespace DancingGoat.Infrastructure
 {
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
     public class SelfConfigActionFilterAttribute : ActionFilterAttribute
```

Comment:
if I remove projectid from appconfig, it should display a configuration dialog. however, it throws an exeption

Reply:
It seems like a configuration problem, I will create the fallback.